### DISCUSSION TITLE
Comparing samples without clustering (cell types are known already)

### DISCUSSION THREAD

**Discussion by benedictuttley** [stuart-lab/signac#483] [Q&A] [answered]
Hi,

I have 3 samples composed of the same cell type but are from different conditions and want to compare the chromatin accessibility between them. I'm especially interested in viewing the `CoveragePlot`.

I am not interested in finding specific cell types, rather just comparing the samples themselves.

I currently plan on emerging the objects (based on the merge vignette) and maybe setting the Idents to be the sample names?
What approach would you recommend? Any help would be much appreciated.

Thanks,
Benedict

**Comment by timoast** [comment, marked as answer]
Yes, I would recommend merging the different objects and setting the idents to the dataset of origin
